util/check_characters.py

```diff
@@ -19,11 +19,14 @@
 import os
 import sys
 
+import unicodedata
+
 def main():
     parser = argparse.ArgumentParser()
 
     parser.add_argument("-csv", "--csv-files", help="Str. Filenames as a comma separated list", required=True)
     parser.add_argument("-alpha", "--alphabet-format", help="Bool. Print in format for alphabet.txt", action="store_true")
+    parser.add_argument("-unicode", "--unicode-variants", help="Bool. Check unicode consistency (use with --alphabet-format)", action="store_true", default=True)
```

Comment:
If you make it `action="store_true"` and also `default=True`, it's impossible to set this flag to False.